lib/logstash/inputs/twitter.rb

```diff
@@ -130,4 +193,33 @@ def from_tweet(tweet)
 
     event
   end
+
+  def configure(c)
+    c.consumer_key = @consumer_key
+    c.consumer_secret = @consumer_secret.value
+    c.access_token = @oauth_token
+    c.access_token_secret = @oauth_token_secret.value
+  end
+
+  def build_options
+    build_options = {}
+    build_options[:track]     = @keywords.join(",")  if @keywords && @keywords.length > 0
```

Comment:
Did you consider using `@keywords.empty?` it is a bit more semantic + readable IMHO. 

Same goes for lines below